native/src/main/java/io/ballerina/lib/wso2/controlplane/ControlPlaneConstants.java

```diff
@@ -7,7 +7,40 @@ public class ControlPlaneConstants {
     public static final String BAL_HOME = "balHome";
```

Comment:
Shall we rename this class to Constants.java.

Reply:
Shall we rename other classes to Artifacts.java, Services.Java and Listeners.Java. Current names are too length IMO